### PR TITLE
Funnel tooltip events

### PR DESCRIPTION
**Pull request by PavelVanecek**
## Description

Same as https://github.com/recharts/recharts/pull/4496 but for another component

## Related Issue

https://github.com/recharts/recharts/discussions/4437

## Motivation and Context

No cloning

## How Has This Been Tested?

npm test

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] I have added a storybook story or extended an existing story to show my changes

### CODE REVIEW

File: src/chart/generateCategoricalChart.tsx [hunk 1]
```diff
@@ -1869,7 +1869,7 @@ export const generateCategoricalChart = ({
 
       if (tooltipEventType !== 'axis' && tooltipItem && tooltipItem.props.trigger === 'click') {
         itemEvents = {
-          onClick: combineEventHandlers(this.handleItemMouseEnter, element.props.onClick),
+          // onClick: combineEventHandlers(this.handleItemMouseEnter, element.props.onClick),
```
Comment (PavelVanecek) [inline src/chart/generateCategoricalChart.tsx:1872]: Oh wait I didn't mean to commit this

Comment (PavelVanecek) [reply src/chart/generateCategoricalChart.tsx:1872]: Tests passed though - if anything this shows a lack of coverage 😢

Comment (ckifer) [reply src/chart/generateCategoricalChart.tsx]: Oops

**Review by ckifer — APPROVED** [review]
Nice nice

**Pull request merged**